content/en/docs/zero-code/java/agent/_index.md

```diff
@@ -5,106 +5,4 @@ aliases:
   - /docs/java/automatic_instrumentation
   - /docs/languages/java/automatic_instrumentation
 redirects: [{ from: /docs/languages/java/automatic/*, to: ':splat' }]
-cSpell:ignore: Dotel myapp
 ---
```

Comment:
This page should have at least a sentence in it, unless you're fully comfortable with it only appearing as a TOC-like page.